.github/workflows/coverity.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         uses: rtCamp/action-slack-notify@v2
         env:
           SLACK_COLOR: 'danger'
-          SLACK_FOOTER:
```

Comment:
This works, but that `null` made me wondering I checked it via GitHub UI

<img width="593" alt="Screenshot 2021-09-24 at 23 00 26" src="https://user-images.githubusercontent.com/22274335/134733138-7aeb53ba-aecc-4f60-b79c-710a75409357.png">

(as it was also a problem according to the built-in syntax checker)

Reply:
The question is, why do we need empty `SLACK_FOOTER`env var?

Reply:
Correct value is probably an explicit empty string. Without wiping it, we get branding crap from the action itself in the Slack messages it sends.

Reply:
Though ironically the ambiguity in these cases between a null value and an empty string kind of highlights one of the reasons to avoid implicit null values...